### PR TITLE
Update GPU docs

### PR DESCRIPTION
**Pull request by sebastian-luna-valero**
# Summary

The `acc-comp.egi.eu` VO was decommissioned and the docs need updating.

This PR updates the relevant page.

---

**Related issue :**

### CODE REVIEW

File: content/en/users/cloud-compute/gpgpu/_index.md [hunk 1]
```diff
@@ -18,14 +18,22 @@ The table below summarises the available options:
...
 | IFCA-LCG2      | up to 2 NVIDIA T4, up to 2 NVIDIA V100 |                                                                                                                                                                                            |                                                            | Pay-per-use                                                                           |
 | CESNET-MCC     | up to 2 NVIDIA T4                      | hpc.8core-64ram-nvidia-1080-glados, hpc.19core-176ram-nvidia-1080-glados, hpc.38core-372ram-nvidia-1080-glados, hpc.19core-176ram-nvidia-2080-glados, hpc.38core-372ram-nvidia-2080-glados | vo.clarin.eu, biomed, eosc-synergy.eu, peachnote.com       | Sponsored, conditions to be negotiated                                                |
 
 <!-- markdownlint-enable line-length -->
 
 ## Access to GPU resources on EGI Cloud
 
+First, you need to make your you belong to one of the supported
```
Comment (gwarf) [inline content/en/users/cloud-compute/gpgpu/_index.md:29]: Nitpicking: when I see something started by `First`, I always look after to see if I will find after things like `Second` (or `Secondly` in fact) and/or `Finally`.

Comment (sebastian-luna-valero) [reply content/en/users/cloud-compute/gpgpu/_index.md]: I agree, thanks!

**Review by gwarf — APPROVED** [review]
LGTM

**Review by enolfc — APPROVED** [review]
👍

**Pull request merged**